src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const act of actions) {
```

Comment:
Destructure `action`
```suggestion
  for (const action of actions) {
    const { type,  keysToRemove, extraData } = action
```